tests/templates.test.js

```diff
@@ -132,8 +132,8 @@ describe('findBestLocaleMatch function', () => {
   });
 });
 
-// Mocking the manifest data
-jest.mock('@/web/templates/manifest.json');
+// // Mocking the manifest data
+// jest.mock('@/web/templates/manifest.json');
```

Comment:
This mock module import was leading to a configuration error. I noticed every other manifest reference in this file was commented out, so I comment the mock import out. Do we need this? The module import error may also have been due to me messing with jest-rollup. Not sure.

Reply:
I'm guessing we don't need it.  I removed all references to manifest.json at begining of month...